naming/src/main/java/com/alibaba/nacos/naming/core/InstanceOperatorServiceImpl.java

```diff
@@ -145,9 +145,6 @@ public void patchInstance(String namespaceId, String serviceName, InstancePatchO
         if (null != patchObject.getHealthy()) {
             instance.setHealthy(patchObject.getHealthy());
         }
-        if (null != patchObject.getApp()) {
```

Comment:
Why?

Reply:
This is a duplicate logic.

same logic in line 139 and 148. So remove 148
```
        if (null != patchObject.getApp()) {
            instance.setApp(patchObject.getApp());
        }
```